doc/classes/RigidBody.xml

```diff
@@ -324,19 +324,19 @@
 	</methods>
 	<members>
 		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp">
-			Dampens rotational forces of the Rigid body by the 'angular_damp' rate.
+			Dampens Rigid Body's rotational forces.
```

Comment:
"RigidBody's" - there's no space in the name.

"Dampen" means making something wet.  "Damp" means reducing a physical quantity.  You "damp" a force.